### PR TITLE
Bring CalendarDatePicker, CheckBox, and ToolTip templates into WinUI

### PR DESCRIPTION
**Pull request by chrisglein**
These templates need to be changed in order to support CornerRadius correctly: (Feature #659)
* CalendarDatePicker
* Checkbox
* ToolTip

These are completely unchanged from what shipped in 19H1, minus some contract7Present wrapping around `x:Load` and `CornerRadius` in `CalendarDatePicker`. I didn't even fix the formatting to create a clean diff... but we'll want to tidy that up.

Also take some existing templates that were under CommonStyles and break into separate projects so there's a natural place to put test pages:
* AutoSuggestBox
* DatePicker
* Slider
* TimePicker

Added test page entry points for all those controls. Test pages are mostly stub-level right now, but I want very low barrier of entry for someone to expand upon them as we work on these in WinUI.

### CODE REVIEW

File: dev/AutoSuggestBox/AutoSuggestBox.vcxitems [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (licanhua) [inline dev/AutoSuggestBox/AutoSuggestBox.vcxitems]: license header for all new added files.

**Comment by licanhua** [comment]
> 

rename CheckBox_rs1_themeresources.xaml to CheckBox_themeresources.xaml

---
Refers to: dev/CheckBox/CheckBox_rs1_themeresources.xaml:1 in 3858c7f. [](commit_id = 3858c7ffa39b78d26fc27b7f3d94016bece32312, deletion_comment = False)

File: dev/AutoSuggestBox/AutoSuggestBox.vcxitems (hunk 1 above)
Comment (chrisglein) [reply dev/AutoSuggestBox/AutoSuggestBox.vcxitems]: I'm not seeing that in other vcxitems. Can you point me to one that has what you're looking for?

File: dev/CheckBox/TestUI/CornerRadiusPage.xaml [hunk 2]
```diff
@@ -0,0 +1,544 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
```
Comment (kaiguo) [inline dev/CheckBox/TestUI/CornerRadiusPage.xaml]: Should this page be in some other folder? CommonStyles?

Comment (chrisglein) [reply dev/CheckBox/TestUI/CornerRadiusPage.xaml]: That's exactly where it's supposed to be, and where I thought I had it. Weird, how'd it get in the Checkbox folder? I'll move it...

Comment (kaiguo) [reply dev/CheckBox/TestUI/CornerRadiusPage.xaml]: I just noticed you've moved it to CommonStyles folder in your other branch user/cglein/migrateCoreControlStyles.  So I guess it's fine to merge this in and let the other PR fix it.

**Pull request merged**